test/reactTags.test.js

```diff
@@ -155,6 +153,22 @@ test("handles the paste event and splits the clipboard on delimiters", () => {
   ]);
 });
 
+test("should not allow duplicate tags", () => {
+  const actual = [];
+  const $el = mount(
+    mockItem({
+      handleAddition(tag) {
+        actual.push(tag);
+      },
+    })
+  );
```

Comment:
Can you add an assertion for checking if 'Apple' is already there?

Reply:
added 👍